apics/static/Atlas/106.html

```diff
@@ -256,8 +256,8 @@ <h4 class="c12" id="section-4afterthefocusedelementandadjacent"><span class="c10
 </div>
 <blockquote class="example" style="font-size:100%;padding-left:1.8em;margin-left:0.3em">
 	
-<p class="c12" style="text-indent:-2em"><span class="c32">(5) <a href="/contributions/48">Palenquero (Schwegler 2013)</a></span></p>
-	<div class="sentence-wrapper">
```

Comment:
It would seem to me that the <p> element should be kept, but rather the redundant `(5)` in the <div> be removed.